mod-login-saml/src/main/resources/domain/mod-login-saml/features/configureSaml.feature

```diff
@@ -0,0 +1,70 @@
+Feature: Configure SAML
+
+  Background:
+    * url baseUrl
+    * call login testAdmin
+    * configure headers =
+    """
+    {
+      "X-Okapi-Tenant": "#(testTenant)",
+      "X-Okapi-Token": "#(okapitoken)",
+      "Accept": "application/json, text/plain"
+    }
+    """
+  Scenario: Validate IdP url
+    Given path "saml/validate"
+    And param type = "idpurl"
+    And param value = idpUrl
+    When method GET
+    Then status 200
```

Comment:
This scenario can be removed; is not needed for this kind of integration test. The GET saml/validate that checks the active flag in line 54 can be kept.

Reply:
Agreed. Will remove.